### ISSUE TITLE
Speeding up loading of the store

### ISSUE THREAD

**Issue by splendirlabs** [OceanLabs/iOS-Print-SDK#263]
Hi,

I'm using the OLKiteViewController (which is awesome btw) to load in the store contents. However, it usually takes 4s+ to load. Is there any recommended way to speed this up?

Is this a production vs. testing env issue maybe? Thx.

**Comment by kkarayannis** [comment]
Hi @splendirlabs 

While it's true that production will be faster, it will still take a few seconds to load. We are constantly working to improve startup times and the next version (v.6.2.0) will improvement things a bit more.

**Comment by splendirlabs** [comment]
@kkarayannis Thanks for the quick reply.

Are there any suggestions to using caching - anything you guys have tried? Thx.

**Comment by lenkawell** [comment]
We've noticed this as well. At first I thought it was just our app, but I've tried PicCollage installed from the App Store and it takes at least 4 seconds or longer on an iPad Pro connected to the internet via WiFi to a 25Mbps connection here in Seattle. For our app we've seen times as long as 10 seconds sometimes. And over a less-than-excellent cell connection it can be worse.

So I'd also like to know if there is anything we can do to make it faster. Is it downloading all the product images each time? Can we disable some products in the Kite.ly dashboard to make it faster? Any other suggestions would be helpful.
